ports/unix/README.md

```diff
@@ -1,14 +1,49 @@
-The Unix version
-----------------
+# The Unix port
```

Comment:
I'm being picky, but most other port's README.md's use `=====` for the top heading, and then `-----` for sub-headings.  Would be good to match them.

Also, most other ports include "MicroPython" in the heading, so I suggest something like "MicroPython Unix port" here.  That would at least match `ports/embed/README.md`.

Reply:
Oh I meant to leave a comment about the header change.

I changed this because I couldn't get the top-level === headings to render as h1. This heading would render as h2, and then the sub-headings would render as h1. I got sick of fiddling with it and swapped to this style.

That's not the case in `ports/esp32` README though, that one renders properly so it's something odd here. I'll try again.

Reply:
Updated and h1/h2 are rendering properly now. :shrug: